packages/backend/server/src/core/doc/storage/index.ts

```diff
@@ -1,4 +1,6 @@
-// TODO(@forehalo): share with frontend
+// This is a totally copy of definitions in [@affine/space-store]
+// because currently importing cross workspace package from [@affine/server] is not yet supported
+// should be kept updated with the original definitions in [@affine/space-store]
```

Comment:
we can use symbolic links in git

Reply:
link will only make things complicated, especially when deploying the wanted package only